io.catenax.pcf/4.0.0/Pcf.ttl

```diff
@@ -0,0 +1,896 @@
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:id a samm:Property ;
+   samm:preferredName "Product Footprint Identifier"@en ;
+   samm:description "The product footprint identifier as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :IdTrait ;
+   samm:exampleValue "3893bb5d-da16-4dc1-9185-11d97476c254" .
+
+:specVersion a samm:Property ;
+   samm:preferredName "Product Footprint Specification Version"@en ;
+   samm:description "Version of the product footprint data specification as defined in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2.0.1-20230314" .
+
+:partialFullPcf a samm:Property ;
+   samm:preferredName "Partial Or Full PCF (Product Carbon Footprint)"@en ;
+   samm:description "Indicator for partial or full PCF (Product Carbon Footprint) declaration as specified in the Catena-X PCF Rulebook."@en ;
+   samm:characteristic :PartialFullPcfCharacteristic ;
+   samm:exampleValue "Cradle-to-gate" .
+
+:precedingPfIds a samm:Property ;
+   samm:preferredName "Previous PCF (Product Carbon Footprint) Identifiers"@en ;
+   samm:description "Set of preceding PCF (Product Carbon Footprint) identifiers without duplicates as specified in the Catena-X PCF Rulebook in accordance with the WBCSD (World Business Council for Sustainable Development) Pathfinder framework and the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD/ PACT initiative. Declared as \"optional\" in WBCSD, needs to be covered by application."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :PrecedingPfIdsTrait .
+
+:version a samm:Property ;
+   samm:preferredName "Product (Carbon) Footprint Version"@en ;
+   samm:description "Version of the product (carbon) footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. In Catena-X set to \"0\" per default."@en ;
```

Comment:
I understood that the models from Cateena-X are about to be standardized within the IDTA. For that they need to be independent of a specific dataspace. So, the reference to Catnea-x n the description might for example need to be rephrased as an example